docs/_chaoss-gmd-cde/pull_request_merge_duration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pull request merge duration
+description: This panel focuses on pull requests merge duration, defined by the time between code merge request and code commit.
+author: Ana Jimenez Santamaria
+created_at: 2019-01-23
+grimoirelab_version: 0.2.3
+layout: panel
+---
+
+
+This panel focuses on **Pull request merge duration**.
+
+
+This panel focuses on pull request merge duration, defined by the time between a code merge request and code commit. In this case, we focus on Github pull request. When we measure Github, we cannot identify if a pull request has been merged, only if is open or closed. That's the reason why we will based our panel on pull request closing time
+
+The main idea behind this panel is to help us identify a project state (growth maturity or decline) by answering custom queries like:
+* **What is the duration of time between code merge request and code commit?**
+* **What organizations/authors are being more efficient when closing a pull request?**
+* **Is my project taking too long when closing code merge requests?**
+
+
+### What should I look for in the panel?
+
+You can see if a pull request is taking too long to be merged by looking at gauge visualization. You can set up time ranges to help you visualize different scenarions (e.g fast, medium or slow response)
```

Comment:
Not sure if this is true. I'd say we can visualize time to close in that specific time frame. For instance, how good or bad we were closing pull request created last year, two years ago or last week.